Handbook.md

```diff
@@ -38,9 +37,27 @@ __Links:__
 __Links:__
 ### 3.1 Topics introduction
 ### 3.2 Important guidelines
-(Write in bullet points)
+* Easy team integration
+* Increased code quality efficiency and easy for maintaining
+* Reduce code complexity
+* Reduce development cost
+
+* Naming Convention
+* File and folder Naming and Organization
+* Formatiting and Indentation
+* Commenting and Documenting
+* Classes and Functions
+* Testing
 ### 3.3 Topics
-(Note that it's just as important to highlight bad practices to avoid as good practices to follow. Each section should also contain links to the reference articles for further reading.)
+The developer's world has too much coding standards. It is important in a software development to select the better one. This coding standards depends on the language used i.e., in C++ you will not use the same coding standard than if the language will be in Java or in PHP. It will also depend on the company. You will have three best options, the easiest option is to follow a coding standard which already exists, another way will be to follow a coding standard and make small change because it does not feat with your company rules. The last option will be to create your own coding standards.
+
+The most difficults for a developeur is to follow or work with another developer who does not use the same coding standards. During the development you must create new folders and files. It is important to use the same named and structured format to lose less time as possible when you are looking for a specific file. i.e an asset like a picture or a font.
+Read code from someone else can be hard when the nameming conventions is not the same. For exemple all class names must be in PascalCase and the variables must be in camelCase. If you use an opposed standards convention for the nameming the other person could not understand really fast what your code is supposed to do. Our team will lose time and the development cost will be increased.
+When you are working in team on the same project an important part is the testing. Two co-working could be working on the same features and without testing the other co-worker could broke your part without knowing what he is doing. The testing has this role to inssure no one is broking your work with additional code.
+All developeurs do not have the same abilities or the same competencies. It is important to commenting and documenting. It will make it easy for the other co-working to understand what is appening in your class, your function, or your package. Without commenting the developer must read and understant the whole concept to work with it. Again, it will increase the time so the development cost.
```

Comment:
typo
developeurs -> developers
co-working -> co-worker
appening -> happening

grammar
commenting -> comment
documenting -> document

```suggestion
All developers do not have the same abilities or the same skills. It is important to comment and document. It will make it easy for the other co-workers to understand what is happening in your class, your function or your package. Without comments, the developer must read and understand the whole concept to work with it. Again, it will increase the time so the development cost.
```